frontend/src/components/pages/CampCreation/index.tsx

```diff
@@ -239,7 +328,11 @@ const CampCreationPage = (): React.ReactElement => {
       case CampCreationPages.ScheduleSessionsPage:
         return (
           <ScheduleSessions
-            campTitle="Waterloo Photography Camp 2022 @7:00 AM - 3:00PM"
+            campTitle={
+              campToEdit
+                ? `${campToEdit.name} @${campToEdit.startTime} - ${campToEdit.endTime}`
+                : "Waterloo Photography Camp 2022 @7:00 AM - 3:00PM"
```

Comment:
I think I can actually just use the campName and states that we defined. On camp edit, the states will be populated with the initial data anyways. Also, the campName, startTime, and endTime will always be non zero values because you can't go to the session scheduling step without filling out the first step